src/utils/country-helpers.js

```diff
@@ -12,6 +12,12 @@ function getCountryInfo( tzName ) {
    try {
         const zoneName = tzName?.trim();
         if( zoneName ) {
+            if(tzName === "UTC"){
```

Comment:
`zoneName` should be checked rather `tzName` as it might have space. Or else we should use `String.includes`

Reply:
Done.